editor/src/components/editor/variablesmenu.spec.browser2.tsx

```diff
@@ -162,6 +162,149 @@ describe('variables menu', () => {
         ),
       )
     })
+    it('shows and does not insert scoped properties when not possible', async () => {
+      const editor = await renderTestEditorWithProjectContent(
+        makeMappingFunctionTestProjectContents(),
+        'await-first-dom-report',
+      )
+
+      await selectComponentsForTest(editor, [
+        EP.fromString(
+          `${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:container/676/020~~~1`,
+        ),
+      ])
+
+      await openVariablesMenu(editor)
+
+      document.execCommand('insertText', false, 'value')
+      expect(getInsertItems().length).toEqual(1)
+      expect(getInsertItems()[0].innerText).toEqual('value')
+
+      const filterBox = await screen.findByTestId(InsertMenuFilterTestId)
+      forceNotNull('the filter box must not be null', filterBox)
+
+      await pressKey('Enter', { targetElement: filterBox })
```

Comment:
I tried the feature branch and the code also shows a toast 'Cannot find a suitable parent', I think we could also expect that here!